operator/go.mod

```diff
@@ -449,6 +449,7 @@ replace (
 	github.com/fluxcd/helm-controller/shim => github.com/redpanda-data/flux-controller-shim/helm/shim v0.0.0-20231227162419-a45126310240
 	github.com/fluxcd/source-controller/shim => github.com/redpanda-data/flux-controller-shim/source/shim v0.0.0-20240113100428-5e301ef97b19
 	github.com/opencontainers/go-digest => github.com/opencontainers/go-digest v1.0.1-0.20230815154656-802ce17c4f59
+	github.com/redpanda-data/common-go/rpadmin => ../../common-go/rpadmin
```

Comment:
remove before merging please (just in case you are as forgetful as I am)

Reply:
👍 that's why I marked this WIP 😅